### PR TITLE
Docs - Snyk Webhook

### PR DESCRIPTION
**Pull request by PeyGis**
# Description

A webhook integration between Port and Snyk to ingest vulnerability entities to Port.

## Added docs pages

/build-your-software-catalog/sync-data-to-catalog/webhook/examples/snyk

### CODE REVIEW

**Comment by PeyGis** [comment]
image

image

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/snyk.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<SnykConfiguration/>
+
+</details>
+
+## Create the Snyk webhook
+
+1. Go to [Synk](https://snyk.io/) and select an account you want to configure the webhook for;
+2. Click on **Settings** at the left of the page and copy your organization ID under the **Organization ID** section;
+3. Navigate to your [Snyk accounts page](https://snyk.io/account/) and copy your API token. You will use this value to authorize the REST API;
+4. Open any REST API client such as POSTMAN and make the following API call to create your webhook:
```
Comment (danielsinai) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/snyk.md:38]: Provide here a `curl` script in addition

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/resources/snyk/_example_snyk_vulnerability_webhook_configuration.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      "entity": {
+        "identifier": ".body.project.id",
+        "title": ".body.project.name + \"-\" + .body.project.origin",
+        "properties": {
+          "organizationUrl": ".body.org.url",
+          "organizationName": ".body.org.name",
+          "projectName": ".body.project.name",
+          "projectOrigin": ".body.project.origin",
+          "branchName": ".body.project.branch",
+          "issueType": ".body.newIssues[0].issueType",
```
Comment (danielsinai) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/resources/snyk/_example_snyk_vulnerability_webhook_configuration.mdx:19]: There could be more than 1 issue in a single webhook?

Comment (PeyGis) [reply docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/resources/snyk/_example_snyk_vulnerability_webhook_configuration.mdx:19]: Yes, there could be more than 1 issue

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/snyk.md (hunk 1 above)
Comment (danielsinai) [reply docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/snyk.md:38]:
```
curl -X POST \
     -H "Authorization: token YOUR_API_KEY" \
     -H "Content-Type: application/json" \
     -d '{"url": "https://ingest.getport.io/YOUR_PORT_WEBHOOK_KEY", "secret": "WEBHOOK_SECRET"}' \
     https://api.snyk.io/v1/org/YOUR_ORG_ID/webhooks
```

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/resources/snyk/_example_snyk_vulnerability_webhook_configuration.mdx (hunk 2 above)
Comment (danielsinai) [reply docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/resources/snyk/_example_snyk_vulnerability_webhook_configuration.mdx:19]: I see thanks! we will wait for the feature in port side then :)

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/snyk.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+   ```json
+   {
+     "url": "https://ingest.getport.io/<YOUR_PORT_WEBHOOK_KEY>",
+     "secret": "WEBHOOK_SECRET"
+   }
+   ```
+5. Click **Send** to create your Snyk webhook;
+6. You can also create the Snyk webhook using the `curl` command below:
+
+```curl showLineNumbers
```
Comment (danielsinai) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/snyk.md:52]: Can you input it in a :::note

**Review by danielsinai — CHANGES_REQUESTED** [review]
🦢 Great job man, small comment :)

**Review by danielsinai — APPROVED** [review]
🦢 Great job!

**Pull request merged**